src/hotspot/share/cds/archiveBuilder.cpp

```diff
@@ -47,8 +47,8 @@
 #include "logging/log.hpp"
 #include "logging/logStream.hpp"
 #include "memory/allStatic.hpp"
-#include "memory/memoryReserver.hpp"
 #include "memory/memRegion.hpp"
+#include "memory/memoryReserver.hpp"
```

Comment:
I think this can be argued is a bug. In previous discussion we (or, at least I) have argued that the sort order should be case-insensitive.